rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -25,34 +25,38 @@ THE SOFTWARE.
 #include "rocal_api_types.h"
 
 /*!
- * \brief Rearranges the order of the frames in the sequences with respect to new_order.
- * Frames can be repeated or dropped in the new_order.
+ * \file
+ * \brief The AMD rocAL Library - Augmentations.
+ *
+ * \defgroup group_rocal_augmentations API: AMD rocAL - Augmentation API
+ * \brief The AMD rocAL augmentation functions.
+ */
+
+/*!
+ * \brief Rearranges the order of the frames in the sequences with respect to new_order. new_order can have values in the range [0, sequence_length). Frames can be repeated or dropped in the new_order.
  * \ingroup group_rocal_augmentations
  * \note Accepts U8 and RGB24 input.
  * \param [in] p_context context for the pipeline.
  * \param [in] p_input Input Rocal Tensor
- * \param [in] new_order represents the new order of the frames in the sequence in the range [0, sequence_length)
- * \param [in] is_output True: the output tensor is needed by user and will be copied to output buffers using the data
- * transfer API calls. False: the output tensor is just an intermediate tensor, user is not interested in
- * using it directly. This option allows certain optimizations to be achieved.
+ * \param [in] new_order represents the new order of the frames in the sequence
```

Comment:
PLease mention "vector of type unsinged int epresenting"

Reply:
Done